### PR TITLE
fix(about): active and hover styles

### PR DESCRIPTION
**Pull request by Fdom92**
https://openforge.teamwork.com/#/tasks/17495250

### CODE REVIEW

File: src/pages/app-about/app-about.scss [hunk 1]
```diff
@@ -124,6 +124,18 @@
       max-width: 420px;
       text-align: left;
     }
+
+    button {
+      &:hover {
+        background: #ff5d00;
```
Comment (PaulPaulDevelops) [inline src/pages/app-about/app-about.scss]: We should use the global variable for this hexcode to keep it consistent.

File: src/pages/app-about/app-about.scss [hunk 2]
```diff
@@ -124,6 +124,18 @@
...
+        border-color: white;
+      }
+
+      &:not(:disabled):active {
+        background-color: #ff5d00;
```
Comment (PaulPaulDevelops) [inline src/pages/app-about/app-about.scss]: We should use the global variable for this hexcode to keep it consistent.

File: src/pages/app-about/app-about.scss (hunk 1 above)
Comment (PaulPaulDevelops) [inline src/pages/app-about/app-about.scss]: We should use the global variable for this hexcode to keep it consistent.

File: src/pages/app-about/app-about.scss [hunk 3]
```diff
@@ -124,6 +124,18 @@
...
+        border-color: white;
```
Comment (PaulPaulDevelops) [inline src/pages/app-about/app-about.scss]: There is a global variable for white, we should use that instead incase we ever need to adjust the shade of it at any point.

File: src/pages/app-about/app-about.scss [hunk 4]
```diff
@@ -124,6 +124,18 @@
       max-width: 420px;
       text-align: left;
     }
+
+    button {
+      &:hover {
+        background: #ff5d00;
+        border-color: white;
```
Comment (PaulPaulDevelops) [inline src/pages/app-about/app-about.scss]: There is a global variable for white, we should use that instead incase we ever need to adjust the shade of it at any point.

File: src/pages/app-about/app-about.scss (hunk 2 above)
Comment (Fdom92) [reply src/pages/app-about/app-about.scss]: you are right, fixed

File: src/pages/app-about/app-about.scss (hunk 1 above)
Comment (Fdom92) [reply src/pages/app-about/app-about.scss]: you are right, fixed

File: src/pages/app-about/app-about.scss (hunk 3 above)
Comment (Fdom92) [reply src/pages/app-about/app-about.scss]: you are right, fixed

File: src/pages/app-about/app-about.scss (hunk 4 above)
Comment (Fdom92) [reply src/pages/app-about/app-about.scss]: you are right, fixed

**Comment by Fdom92** [comment]
@Wholloran @PaulPaulDevelops updated and fixed

**Pull request merged**